content/en/bits_ai/bits_ai_dev_agent/setup.md

```diff
@@ -7,38 +7,30 @@ disable_toc: false
 
 Bits AI Dev Agent integrates with GitHub to open, update, and iterate on pull requests based on issues detected in Datadog.  
 
-<div class="alert alert-info">Bits AI Dev Agent supports GitHub only.</div>
-
 ## Setup
 
-The following steps are required to get started with Bits AI Dev Agent. 
-
-### Step 1: Enable the GitHub integration
-
-Install the [GitHub integration][5]. For full installation and configuration steps, see the [GitHub integration guide][6].
-
-### Step 2: Configure GitHub permissions
+1. Install the [GitHub integration][5]. For full installation and configuration steps, see the [GitHub integration guide][6].
 
-The GitHub integration must be granted the following permissions to enable basic Dev Agent functionality:
+1. Configure GitHub permissions:
 
-- **Repository Permissions**
-  - `Contents: Read & Write`
-  - `Pull Requests: Read & Write`
-- **Subscribe to Events**
-  - `Push`
+   1. To enable basic Dev Agent functionality, configure the GitHub integration with the following permissions:
 
-#### Additional permissions for CI integration
+      - **Repository Permissions**
+        - `Contents: Read & Write`
+        - `Pull Requests: Read & Write`
+      - **Subscribe to Events**
+        - `Push`
 
-To allow the Dev Agent to use CI logs when iterating on pull requests, you must send CI logs to Datadog and enable the [auto-push](#enable-auto-push) feature. This requires additional permissions:  
+    1. (Optional) To allow the Dev Agent to use CI logs when iterating on pull requests, you must send CI logs to Datadog and enable the [auto-push](#enable-auto-push) feature. This requires additional permissions:  
 
-- **Repository permissions**  
-  - `Checks: Read`  
-  - `Commit statuses: Read only`  
-- **Subscribe to events**  
-  - `Check run`  
-  - `Check suite`  
-  - `Issue comment`  
-  - `Status` 
+        - **Repository permissions**  
+          - `Checks: Read`  
+          - `Commit statuses: Read only`  
+        - **Subscribe to events**  
+          - `Check run`  
+          - `Check suite`  
+          - `Issue comment`  
+          - `Status` 
```

Comment:
Only `issue comments` is listed out of these four on the Events tab. Not sure if that means that the tab should be updated? Or maybe I'm looking in the wrong place?